script/Arcueid/Day 9/09_00_ARC09_1_1.txt

```diff
@@ -55,6 +56,6 @@ The one thing that's bothering me is that reaction.
 [sha:1cf6cb3cadf66fbc02cfe87050dd25ecbad57f13]{
 -- Page 17, Offset 21359.
 -- 傷が治りきっていないのに暴れた反動だろう。今夜あたり、ちゃんと完治しているか確かめてやる。
-I'm sure she's just paying the price for being so rambunctious even though her wounds haven't healed yet.
- When I get the chance tonight I'll try and confirm if she really has fully recovered or not.
+I'm sure she's just paying the price for being so rambunctious, even though her wounds haven't healed yet.
```

Comment:
Not sure I agree with the comma here, to me it partitions the sentence such that "even though her wounds haven't healed yet" sounds like the state of _not_ being healed would make her _not_ pay the price.

Reply:
Right. "rambunctious + not having wounds healed" is what she's paying the price for. Not "(paying price for being rambunctious) + wounds havent healed"